Formula/minetest.rb

```diff
@@ -65,4 +64,14 @@ def caveats
       "/Applications/minetest.app/Contents/MacOS/minetest --server".
     EOS
   end
+    
+  test do
+    #
+    # Commands --help, --version will produce a readable output without creating a file.
+    #
+    # Commands --info, --run-unittests, --speedtests, --trace, and
+    # --videomodes require creating user data directory for output.
```

Comment:
We don't really mind creating a user datadir. In our test sandbox, the datadir will be erased after the test is run anyway.

Reply:
Okay.

I'll have to do a bit of reading in the documentation on how to do that. Maybe, by the next update I'll have figured it out.

Reply:
I've changed the file and got four more modes working.

Reply:
Scratch that. Irrlicht Engine reporting No doublebuffering available.
That leaves only one option.